### ISSUE TITLE
UI - It should be possible to change the font size and style

### ISSUE THREAD

**Issue by ThePettsoN** [NilssonPontusOrg/LFGAnnouncements#3]
To improve accessibility it should be possible to change font size & style.

DoD:
- It's possible to change the font style from the settings UI
- It's possible to change the font size from the settings UI
- Font style & size are being saved in the profile data
